rust/src/iexplorer.rs

```diff
@@ -0,0 +1,115 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+use std::error::Error;
+use std::path::PathBuf;
+
+use crate::downloads::read_redirect_from_link;
+use crate::files::compose_driver_path_in_cache;
+
+use crate::manager::{get_minor_version, BrowserManager};
+
+use crate::metadata::{
+    create_driver_metadata, get_driver_version_from_metadata, get_metadata, write_metadata,
+};
+
+const BROWSER_NAME: &str = "iexplorer";
+const DRIVER_NAME: &str = "IEDriverServer";
+const DRIVER_URL: &str = "https://github.com/SeleniumHQ/selenium/releases/";
+const LATEST_RELEASE: &str = "latest";
```

Comment:
As discussed offline. We probably need to build a metadata file with this information?

Reply:
IMO, yes, it is a good idea to publish the latest version of the published assets. This info can be used for managers (Selenium Manager and maybe others) to discover the latest IEDriverServer version easily. @titusfortner created an issue for this: #11280